lib/benchmarks/exe/restore-bench.hs

```diff
@@ -204,11 +204,13 @@ import Cardano.Wallet.Primitive.Types
     , SortOrder (..)
     , WalletId (..)
     , WalletName (..)
-    , chainPointFromBlockHeader
     )
 import Cardano.Wallet.Primitive.Types.Address
     ( Address (..)
     )
+import Cardano.Wallet.Primitive.Types.Block
+    ( chainPointFromBlockHeader'
```

Comment:
can't you just now stick to the name `chainPointFromBlockHeader` (ie. without tick) ? :thinking:

Reply:
I'm afraid not yet — there are two types `ChainPoint` and `Read.ChainPoint` now, and there are going to stay this way for a while as I don't want to invest the effort of refactoring `Cardano.Wallet` all the way up to the database yet.

I hope to be able to remove the `chainPointFromBlockHeader'` function again at a later time, but not in this pull request.